moodle-scalable-cluster-ubuntu/README.md

```diff
@@ -0,0 +1,41 @@
+# MoodleAzure
+Moodle deployment using Azure Resource Manager Template
+
```

Comment:
It would be good to outline the advantages/differences of this template vs existing moodle templates in README

https://github.com/Azure/azure-quickstart-templates/tree/master/moodle-cluster-centos
https://github.com/Azure/azure-quickstart-templates/tree/master/moodle-cluster-ubuntu